docs/hooks/collections.mdx

```diff
@@ -281,9 +281,9 @@ const afterRefreshHook: CollectionAfterRefreshHook = async ({
 For auth-enabled Collections, this hook runs after `me` operations.
 
 ```ts
-import { CollectionAfterMeHook } from 'payload/types';
+import { CollectionMeHook } from 'payload/types';
```

Comment:
This is not correct. The `CollectionMeHook` type is for entirely different hook. It should be this:
```suggestion
import { AfterMe } from 'payload/types';
```

Coming from here: https://github.com/payloadcms/payload/blob/4d0befb67a9cf2d368f4866ccf87f4671c26b527/packages/payload/src/collections/config/types.ts#L190

If its not exported yet, it should be.